### PR TITLE
feat: utilities to count I-D submitters and authors by year

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: ietf/utils/tests_reports.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+from ietf.submit.factories import SubmissionFactory
+from ietf.utils.reports import authors_by_year, submitters_by_year, unique_people
+from ietf.utils.test_utils import TestCase
+
+
+class ReportTests(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        # Build 5 drafts submitted across two years, with 6 unique authors,
```
Comment (jennifer-richards) [inline ietf/utils/tests_reports.py:19]: I count only 5 authors: I don't see an author2 here. Fix comment?

**Review by jennifer-richards — APPROVED** [review]
Looks good to me. I have a few comments/suggestions, but they're all minor.

File: ietf/utils/tests_reports.py (hunk 1 above)
Comment (rjsparks) [reply ietf/utils/tests_reports.py:19]: author0-author5 == 6 authors?

**Review by jennifer-richards — DISMISSED** [review]
Looks good to me

**Comment by rjsparks** [comment]
closing this as asleep - it can be reopened if needed.

**Pull request closed without merging**